<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuildSettings</class>
 <widget class="QDialog" name="BuildSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Build Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the build profile for the current project:</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>clone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>delete</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>new</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Profile Details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0" colspan="4">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Compiler command for *.c files: (avr-gcc)</string>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="QLineEdit" name="compilerC"/>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Compiler command for *.cpp, *.ino, *.pde files: (avr-g++)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLineEdit" name="compilerCPP"/>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Archive command for core.a: (avr-ar)</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QLineEdit" name="coreA"/>
      </item>
      <item row="8" column="0" colspan="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Linking command: (avr-gcc)</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="4">
       <widget class="QLineEdit" name="linker"/>
      </item>
      <item row="14" column="0" colspan="4">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Path to arduino core: (.../cores/arduino/)</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="3">
       <widget class="QLineEdit" name="pathCore"/>
      </item>
      <item row="0" column="0" colspan="4">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Profile Name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QLineEdit" name="profileName">
        <property name="inputMask">
         <string/>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="16" column="0" colspan="4">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Path to AVR includes:</string>
        </property>
       </widget>
      </item>
      <item row="17" column="0" colspan="3">
       <widget class="QLineEdit" name="pathIncludes"/>
      </item>
      <item row="15" column="3">
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="17" column="3">
       <widget class="QToolButton" name="toolButton_2">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="4">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Save Profile</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Object copy command: (avr-objcopy)</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="4">
       <widget class="QLineEdit" name="objCopy"/>
      </item>
      <item row="12" column="0" colspan="4">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Upload command USB: (avrdude)</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0" colspan="4">
       <widget class="QLineEdit" name="uploader"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>BuildSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>BuildSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
